packages/suite/src/views/onboarding/steps/SecurityCheck/SecurityCheck.tsx

```diff
@@ -69,18 +68,8 @@ const Buttons = styled.div<{ $elevation: Elevation }>`
     display: flex;
     flex-wrap: wrap;
```

Comment:
You probably need to change also flex wrapping here. It should be in media query where`flex-wrap: wrap` is set only for small resolutions.